napari/plugins/_npe2.py

```diff
@@ -131,9 +131,13 @@ def populate_qmenu(menu: QMenu, menu_key: str):
             subm = menu.addMenu(subm_contrib.label)
             populate_qmenu(subm, subm_contrib.id)
         else:
-            cmd = pm.get_command(item.command)
-            action = menu.addAction(cmd.title)
-            action.triggered.connect(lambda *args: cmd.exec(args=args))
+            try:
+                cmd = pm.get_command(item.command)
+                action = menu.addAction(cmd.title)
+                action.triggered.connect(lambda *args: cmd.exec(args=args))
+            except KeyError:
+                # this does not have a command associated with it.
+                pass
 
```

Comment:
I think I had a problem here when I was writing tests using the sample plugin fixture.  There was not a command associated with the item.  I don't know if this is something that should be allowed.   If you are going to put it in a menu, should it have a command?  Should we put a warning here if one is not found?

Reply:
see the schema [here](https://github.com/napari/npe2/blob/7791e1016641a6193e91e06b418db5273a678652/npe2/manifest/menus.py#L45-L50).  All menu items that are not submenus must have a command